Pipfile

```diff
@@ -8,6 +8,7 @@ bottle = "*"
 pip = "==18.0"
 pre-commit = "*"
 black = "*"
+pytest = "*"
```

Comment:
this goes inside dev packages. `pipenv install pytest --dev`

Reply:
i used
`pipenv install pytest`
`pipenv lock --dev`
That doesn't work?

Reply:
And yes. The HTML implementation works.